src/modules/Likes/addLikes.js

```diff
@@ -0,0 +1,18 @@
+const addLikes = async (idMeal) => {
+  const likeUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/jcTX0b7AP0QeAh1m307k/likes';
+  const newLike = {
+    item_id: idMeal,
+  };
+  const response = await fetch(likeUrl, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(newLike),
+  });
+  if (response.ok) {
+    window.location.reload();
```

Comment:
after adding a new like it reloads the app every time for each like. may be it will be good if it only updates the targeted element since reloading needs time to fetch from API

Reply:
ok. that improves the speed. on it